crates/node/src/migration_service/web.rs

```diff
@@ -119,6 +130,25 @@ mod tests {
         assert!(res.is_err());
     }
 
+    fn wait_for_keyshare(
```

Comment:
this function is not needed, because we can directly import into the keyshare storage.